sdk/dapp-kit/test/hooks/useWallet.test.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
I think testing the hook in this fashion makes more sense with `renderHook` as opposed to doing the basically equivalent thing in a component test file. I didn't write any additional tests for `WalletProvider` because it's implicitly tested here